src/registrar/tests/test_admin.py

```diff
@@ -243,8 +263,11 @@ def test_place_and_remove_hold_epp(self):
         raise
 
     def tearDown(self):
-        User.objects.all().delete()
         super().tearDown()
+        Domain.objects.all().delete()
+        DomainInformation.objects.all().delete()
+        DomainApplication.objects.all().delete()
+        User.objects.all().delete()
```

Comment:
(Unrelated to your PR / Informational) I don't think it concerns this test but just in general, when your dealing with deleting domains (and by extension DomainInformation) I've noticed that if you have any lingering PublicContact objects, you can't do this directly as you need to delete those contacts first, from a table perspective. 

If this class ever (in one of the test cases), for some reason, wants to test on a domain with PublicContacts this teardown _will_ fail (security.gov / freeman.gov + MockEppLib inherit). That said, we aren't and I do not think this is an issue worth addressing.